_topic_maps/_topic_map.yml

```diff
@@ -19,3 +19,10 @@ Distros: openshift-service-mesh
 Topics:
 - Name: Updating OpenShift Service Mesh
   File: ossm-updating-openshift-service-mesh
+---
+Name: Observability
+Dir: observability
+Distros: openshift-service-mesh
+Topics:
+- Name: Metrics, logs, traces
```

Comment:
I think long term it might be good to separate these into their own topics since users will configure these separately and they are independent of one another. Users may want to configure metrics but not traces or metrics + Kiali but not traces. Separate topics might look like:
```
- Metrics
- Traces
- Logs
```
I think Kiali could even be it's own topic. Something like:
```
- Metrics
- Traces
- Logs
- Kiali
```

Reply:
Ah I see you've already addressed this: https://github.com/openshift/openshift-docs/pull/80434/files#diff-37e6e97f3990f254fa9429eed5272f9251a744bd8c4e39b8061a2ed4d4b0ec67R13

Reply:
The information architecture is fluid, and this suggestion is more for GA than TP 1.